documentation/src/docs/security/pravega-security-configurations.md

```diff
@@ -0,0 +1,44 @@
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+#Pravega Security Configurations
```

Comment:
If you click on the "View" button above, you'll see that this table is not properly formatting.

Reply:
Fixed.